src/main/kotlin/com/mparticle/kits/AdjustKit.kt

```diff
@@ -115,8 +119,9 @@ class AdjustKit : KitIntegration(), OnAttributionChangedListener, ActivityLifecy
     }
 
     companion object {
-        private const val APP_TOKEN = "appToken"
+        const val APP_TOKEN = "appToken"
```

Comment:
private const val APP_TOKEN = "appToken"
Please keep this as private only, we can still access in unit testing.